content/api_chef_server.md

```diff
@@ -6533,6 +6533,71 @@ same name as a user. The response for a user or client is similar to:
 </tbody>
 </table>
 
+### /required_recipe
+
+The `/required_recipe` endpoint has the following method: `GET`.
+
+#### GET
+
+The `GET` method is used to get a copy of a recipe specified by an administrator
```

Comment:
```suggestion
Use the `GET` method to view a specific recipe that all Chef Clients that connect to the Chef Server run, beyond the node's run list.  Returnes a single recipe file that is not part of a cookbook. `required_recipe` is an expert-level feature for delivering isolated configuration changes to target systems.
```
I'm confused by "specified by an administrator". Who/what is the administrator in this context?

Reply:
I think I cleared up what I meant in my last push.  Thanks for the comments.